src/benchmarklib/tpcds/tpcds_table_generator.cpp

```diff
@@ -1148,7 +1154,9 @@ std::shared_ptr<Table> TPCDSTableGenerator::generate_web_site(ds_key_t max_rows)
 
   auto web_site = W_WEB_SITE_TBL{};
   static_assert(sizeof(web_site.web_class) == 51);
+  // NOLINTBEGIN(cppcoreguidelines-pro-type-vararg,hicpp-vararg)
```

Comment:
Disable the rule?

Reply:
We only have it in this file (at least with clang-tidy 14).